src/test/groovy/com/palantir/gradle/docker/DockerRunPluginTests.groovy

```diff
@@ -174,12 +175,12 @@ class DockerRunPluginTests extends AbstractPluginTest {
         BuildResult buildResult = with('docker', 'dockerRemoveContainer', 'dockerRun', 'dockerRunStatus').build()
 
         then:
-        buildResult.task(':dockerRemoveContainer').outcome == TaskOutcome.SUCCESS
+        buildResult.task(':dockerRemoveContainer').outcome == SUCCESS
 
-        buildResult.task(':dockerRun').outcome == TaskOutcome.SUCCESS
+        buildResult.task(':dockerRun').outcome == SUCCESS
         buildResult.output =~ /(?m)HELLO WORLD/
-        buildResult.task(':dockerRunStatus').outcome == TaskOutcome.SUCCESS
-        buildResult.output =~ /(?m):dockerRunStatus\nDocker container 'bar' is STOPPED./
+        buildResult.task(':dockerRunStatus').outcome == SUCCESS
+        buildResult.output =~ /(?m):dockerRunStatus\nDocker container 'foo' is STOPPED./
```

Comment:
@markelliot Apologies for that - I tend to refactor test classes as I work to understand them. Happy to revert these style changes if need be.